src/algebra/char_zero.lean

```diff
@@ -5,7 +5,7 @@ Authors: Mario Carneiro
 
 Natural homomorphism from the natural numbers into a monoid with one.
 -/
-import algebra.field
+import algebra.field data.nat.cast
```

Comment:
@semorrison will be very happy if you adhere to 1 import per line.

Reply:
Is that a rule now? I'm happy with either style (all imports in one block, or one import per line) but mixing them bothers me.

Reply:
I think all of mathlib is now 1 per line